### ISSUE TITLE
utm_* fbclid parameter

### ISSUE THREAD

**Issue by slimco** [lochmueller/staticfilecache#209]
Hi, 
I use nginx and, 
do you have a cache solution using with marketing parameters in url? utm_* and fbclid. 

Thank you

**Comment by lochmueller** [comment]
Hey @slimco 
Do you try to remove the utm_* params in your nginx before the request reach TYPO3?
Regards,
Tim

**Comment by slimco** [comment]
I dont remove utms params but google analytics on site dont catch params.

po 28. 10. 2019 o 15:14 Tim <notifications@github.com> napísal(a):

**Comment by lochmueller** [comment]
Hmm... I don't understand your answer. The arguments are only evaluated by the browser/js, right? So you can drop the arguments for backend calls to TYPO3?

**Comment by lochmueller** [comment]
Hey @slimco 
I close this isse. Arguments cannot be cached by StaticFileCache. It could be part of the FallbackMiddleware but not of the fast hatcess caching rule. It is possible but a huge amount of work to get a configureable and stable htacess rule set.
Other option could be a reverse proxy in front of the typo3 server that drop the arguments, because the arguments are used by the frontend JS they are useless in the TYPO3 request.
Regards,
Tim
